src/css/carousel.css

```diff
@@ -21,18 +27,24 @@
   font-size: 2.5rem;
 }
 
+/* .alice-carousel__prev-btn,
+.alice-carousel__next-btn {
+  padding: 0;
+  margin: 0;
+} */
+
 html[data-theme='dark'] .alice-carousel__dots {
-    margin: 16px 4px 4px;
+  margin: 0;
 }
 
 html[data-theme='light'] .alice-carousel__dots {
-    margin: 16px 4px 4px;
+  margin: 0;
 }
 
 html[data-theme='dark'] .alice-carousel__next-btn [data-area]::after {
   color: white;
 }
 
-html[data-theme='dark'] .alice-carousel__prev-btn [data-area]::after{
+html[data-theme='dark'] .alice-carousel__prev-btn [data-area]::after {
   color: white;
 }
```

Comment:
**Is it possible to add a bit more space between the icons?**

<img width="505" alt="Screen Shot 2022-05-11 at 11 59 37 AM" src="https://user-images.githubusercontent.com/56411016/167926184-ce03d625-ccee-4385-9029-eb3e83c8fe1d.png">

Reply:
I will replace this library for a "homemade" component to have more control over it :D 
Right now, if I change the space, it cuts the image in different screen sizes o_o hopefully I can create it fast